spec/services/csv_upload_validator_spec.rb

```diff
@@ -3,31 +3,59 @@
 require "rails_helper"
 
 RSpec.describe CsvUploadValidator do
+  let(:catalog_record_id_header) { CatalogRecordId.label.downcase.tr(" ", "_") }
+  let(:csv) { "Druid,#{catalog_record_id_header},Barcode\ndruid:bb396kf5077,13157971,\ndruid:bb631ry3167,13965062,\n" }
+  let(:headers) { ["Druid", catalog_record_id_header] }
   let(:validator) { described_class.new(csv:, headers:) }
 
-  let(:csv) { "Druid,Catkey,Barcode\ndruid:bb396kf5077,13157971,\ndruid:bb631ry3167,13965062,\n" }
```

Comment:
Somehow we were still hard-coding this instead of switching the value based on the Folio feature flag. 🤷🏻‍♂️